README.md

```diff
@@ -51,6 +51,15 @@ quite compatible. Code from *Node.js* can be easily rewritten from JavaScript in
  - [OpenSSL][7] 
  - [JSON4Lua][9]
  - [lunit][10]
+
+### Ubuntu 12.04 Precise
+ ```bash
+ sudo apt-get install lua5.1 liblua5.1-0-dev liblua5.1-json libssl-dev libboost1.46-dev libboost-system1.46-dev luarocks cmake libboost-date-time1.48.0 libboost-date-time1.48-dev libboost-thread1.48.0 libboost-thread1.48-dev libboost-system1.48.0 libboost-system1.48-dev

```

Comment:
Thanks for the consideration, I wrote this PR under misguided assumptions. Sorry for the noise.